shortcircuit-api/src/main/java/com/powsybl/shortcircuit/converter/FaultResultDeserializer.java

```diff
@@ -52,6 +54,14 @@ public FaultResult deserialize(JsonParser parser, DeserializationContext deseria
                     threePhaseFaultCurrent = parser.readValueAs(Float.class);
                     break;
 
+                case "contributionsResult":
+                    while (parser.nextToken() != JsonToken.END_OBJECT) {
```

Comment:
we'll need to add a unit test for testing the serialization/deserialization.
We can deserialize as a List instead of iterating explicitly.